pom.xml

```diff
@@ -50,6 +50,12 @@
             <version>2.2.4</version>
         </dependency>
         <dependency>
+            <groupId>junit</groupId>
+            <artifactId>junit</artifactId>
+            <version>4.12</version>
```

Comment:
With this version tests fail with the same issue: https://github.com/spring-io/platform/issues/113
Downgrade to 4.11